WordPress/src/main/java/org/wordpress/android/ui/stats/StatsReferrersFragment.java

```diff
@@ -196,7 +200,7 @@ public View getGroupView(final int groupPosition, boolean isExpanded,
             // totals
             holder.totalsTextView.setText(FormatUtils.formatDecimal(total));
 
-            holder.networkImageView.setImageUrl(PhotonUtils.fixAvatar(icon, mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.STATS_SITE_AVATAR);
+            holder.networkImageView.setImageUrl(GravatarUtils.fixGravatarUrl(icon, mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.BLAVATAR);
```

Comment:
I think here is better to call `GravatarUtils.fixGravatarUrl` with the default picture set to STATUS_404. So when the Gravatar doesn't exist, it returns 404, and networkImageView takes care of defaulting.